diff --git a/node_modules/gatsby-plugin-meta-redirect/gatsby-node.js b/node_modules/gatsby-plugin-meta-redirect/gatsby-node.js
index 51d54b3..74c6422 100644
--- a/node_modules/gatsby-plugin-meta-redirect/gatsby-node.js
+++ b/node_modules/gatsby-plugin-meta-redirect/gatsby-node.js
@@ -7,7 +7,7 @@ async function writeRedirectsFile(redirects, folder, pathPrefix) {
   if (!redirects.length) return;
 
   for (const redirect of redirects) {
-    const { fromPath, toPath } = redirect;
+    const { fromPath, toPath, trailingSlash = true } = redirect;
 
     const FILE_PATH = path.join(
       folder,
@@ -23,7 +23,7 @@ async function writeRedirectsFile(redirects, folder, pathPrefix) {
         // ignore if the directory already exists;
       }
 
-      const data = getMetaRedirect(toPath);
+      const data = getMetaRedirect(toPath, trailingSlash);
       await writeFile(FILE_PATH, data);
     }
   }
diff --git a/node_modules/gatsby-plugin-meta-redirect/getMetaRedirect.js b/node_modules/gatsby-plugin-meta-redirect/getMetaRedirect.js
index ae2545d..f82dece 100644
--- a/node_modules/gatsby-plugin-meta-redirect/getMetaRedirect.js
+++ b/node_modules/gatsby-plugin-meta-redirect/getMetaRedirect.js
@@ -1,4 +1,4 @@
-module.exports = function getMetaRedirect(toPath) {
+module.exports = function getMetaRedirect(toPath, trailingSlash) {
   let url = toPath.trim();
 
   const hasProtocol = url.includes('://');
@@ -10,6 +10,10 @@ module.exports = function getMetaRedirect(toPath) {
 
     const resemblesFile = url.includes('.');
     if (!resemblesFile) {
+      url = url.replace(/\/\/+/g, '/');
+    }
+
+    if (trailingSlash) {
       url = `${url}/`.replace(/\/\/+/g, '/');
     }
   }
