diff --git a/node_modules/path-to-regexp/index.js b/node_modules/path-to-regexp/index.js
index 95d2f4b..a3d4924 100644
--- a/node_modules/path-to-regexp/index.js
+++ b/node_modules/path-to-regexp/index.js
@@ -94,6 +94,7 @@ function pathToRegexp(path, keys, options) {
       pos = offset + match.length;
 
       if (match === '*') {
+        backtrack = '';
         extraOffset += 3;
         return '(.*)';
       }
@@ -123,6 +124,7 @@ function pathToRegexp(path, keys, options) {
         + ')'
         + optional;
 
+      backtrack = '';
       extraOffset += result.length - match.length;
 
       return result;
