diff --git a/node_modules/turndown/lib/turndown.cjs.js b/node_modules/turndown/lib/turndown.cjs.js
index 38ded76..2482d17 100644
--- a/node_modules/turndown/lib/turndown.cjs.js
+++ b/node_modules/turndown/lib/turndown.cjs.js
@@ -15,30 +15,55 @@ function repeat (character, count) {
 }
 
 var blockElements = [
-  'address', 'article', 'aside', 'audio', 'blockquote', 'body', 'canvas',
-  'center', 'dd', 'dir', 'div', 'dl', 'dt', 'fieldset', 'figcaption',
-  'figure', 'footer', 'form', 'frameset', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6',
-  'header', 'hgroup', 'hr', 'html', 'isindex', 'li', 'main', 'menu', 'nav',
-  'noframes', 'noscript', 'ol', 'output', 'p', 'pre', 'section', 'table',
-  'tbody', 'td', 'tfoot', 'th', 'thead', 'tr', 'ul'
-];
+  'ADDRESS', 'ARTICLE', 'ASIDE', 'AUDIO', 'BLOCKQUOTE', 'BODY', 'CANVAS',
+  'CENTER', 'DD', 'DIR', 'DIV', 'DL', 'DT', 'FIELDSET', 'FIGCAPTION', 'FIGURE',
+  'FOOTER', 'FORM', 'FRAMESET', 'H1', 'H2', 'H3', 'H4', 'H5', 'H6', 'HEADER',
+  'HGROUP', 'HR', 'HTML', 'ISINDEX', 'LI', 'MAIN', 'MENU', 'NAV', 'NOFRAMES',
+  'NOSCRIPT', 'OL', 'OUTPUT', 'P', 'PRE', 'SECTION', 'TABLE', 'TBODY', 'TD',
+  'TFOOT', 'TH', 'THEAD', 'TR', 'UL'
+]
 
 function isBlock (node) {
-  return blockElements.indexOf(node.nodeName.toLowerCase()) !== -1
+  return is(node, blockElements);
 }
 
 var voidElements = [
-  'area', 'base', 'br', 'col', 'command', 'embed', 'hr', 'img', 'input',
-  'keygen', 'link', 'meta', 'param', 'source', 'track', 'wbr'
+  'AREA', 'BASE', 'BR', 'COL', 'COMMAND', 'EMBED', 'HR', 'IMG', 'INPUT',
+  'KEYGEN', 'LINK', 'META', 'PARAM', 'SOURCE', 'TRACK', 'WBR'
 ];
 
 function isVoid (node) {
-  return voidElements.indexOf(node.nodeName.toLowerCase()) !== -1
+  return is(node, voidElements);
 }
 
-var voidSelector = voidElements.join();
 function hasVoid (node) {
-  return node.querySelector && node.querySelector(voidSelector)
+  return has(node, voidElements);
+}
+
+var meaningfulWhenBlankElements = [
+  'A', 'TABLE', 'THEAD', 'TBODY', 'TFOOT', 'TH', 'TD', 'IFRAME', 'SCRIPT',
+  'AUDIO', 'VIDEO'
+];
+
+function isMeaningfulWhenBlank (node) {
+  return is(node, meaningfulWhenBlankElements);
+}
+
+function hasMeaningfulWhenBlank (node) {
+  return has(node, meaningfulWhenBlankElements);
+}
+
+function is (node, tagNames) {
+  return tagNames.indexOf(node.nodeName) >= 0;
+}
+
+function has (node, tagNames) {
+  return (
+    node.getElementsByTagName &&
+    tagNames.some(function (tagName) {
+      return node.getElementsByTagName(tagName).length
+    })
+  );
 }
 
 var rules = {};
@@ -589,10 +614,11 @@ function Node (node) {
 
 function isBlank (node) {
   return (
-    ['A', 'TH', 'TD', 'IFRAME', 'SCRIPT', 'AUDIO', 'VIDEO'].indexOf(node.nodeName) === -1 &&
-    /^\s*$/i.test(node.textContent) &&
     !isVoid(node) &&
-    !hasVoid(node)
+    !isMeaningfulWhenBlank(node) &&
+    /^\s*$/i.test(node.textContent) &&
+    !hasVoid(node) &&
+    !hasMeaningfulWhenBlank(node)
   )
 }
 
